sdkproject/Assets/SceneViewBookmarker.cs

```diff
@@ -0,0 +1,178 @@
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace SceneViewBookmarker
```

Comment:
Is this addition part of the styling UI? It seems like it shouldn't be in `Assets`. It could be a good addition to the DevTools though.